docs/diagram-tech-arq.plantuml

```diff
@@ -12,22 +12,26 @@ left to right direction
 
 APIGateway(apiGateway, "API Gateway", "External API Patterns")
 ElasticKubernetesService(creditCrytoService, "Credit Crypto Service", "Saga Pattern")
-Blockchain(blockchainService, "Core Service", "Smart Contract")
+ElasticKubernetesService(blockchainService, "Core", "Block chain client")
```

Comment:
actually this one is a service that communicates with the blockchain, that's the one that knows how to do so